lib/src/facts/linux/filesystem_resolver.cc

```diff
@@ -75,12 +75,6 @@ namespace facter { namespace facts { namespace linux {
         char buffer[4096];
         while (mntent *ptr = getmntent_r(file, &entry, buffer, sizeof(buffer))) {
             string device = ptr->mnt_fsname;
-            string mtype = ptr->mnt_type;
-
-            // Skip over anything that doesn't map to a device and is not tmpfs
-            if (!boost::starts_with(device, "/dev/") && mtype != "tmpfs") {
-                continue;
```

Comment:
Hi,
Thank you for your contribution. This change will display the nfs server mounting point but will display things from /sys, /sys/kernel, /proc and this is not really a good thing as you don't want to be flooded will 20 or more mount points related to the kernel. I think is better to change the "if" statement in order to skip things from /sys and /proc directory.

Reply:
Also, please rebase your changes to 3.11.x not on master.

Reply:
I agree, the noise from /proc and /sys are unintentional consequences of this change. I've pushed a new change and rebased this branch against 3.11.x to remove this noise. Please let me know if there are any other changes you would suggest. Thank you for taking the time to review this.